firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -522,7 +522,7 @@ class StringSetting: Setting, UITextFieldDelegate {
     }
 
     func alignTextFieldToNatural() {
-        textField.textAlignment = .natural
+        textField.textAlignment = textField.effectiveUserInterfaceLayoutDirection == .leftToRight ? .natural : .right
```

Comment:
I'm confused... I thought that `.natural` was supposed to respect RTL device orientation? Why are the other fields in the screen in correct RTL order in the issue screenshot? is it the same cell? Is the alignment possibly not getting called for the last cell?

Reply:
Other fields have labels and this one is a text field. What I found was `.natural` respects keyboard language, not the device language.

Reply:
Huh. TIL! awesome. Tested and it works.